packages/starlight/vitest.config.ts

```diff
@@ -18,12 +18,14 @@ export default defineConfig({
 				'props.ts',
 				// Main integration entrypoint — don’t think we’re able to test this directly currently.
 				'index.ts',
+				// Since Vitest 2.1.2, coverage is collected for `*.astro` files.
+				'**/*.astro',
```

Comment:
Since [Vitest 2.1.2](https://github.com/vitest-dev/vitest/releases/tag/v2.1.2), `*.astro` files are now included in the coverage report by default.